src/less/components/city-chooser.less

```diff
@@ -24,10 +24,11 @@
 
 .city-arm {
   height: 12vw;
+  max-height: 112px;
 }
 
 .city-name {
-  font-family: moon-regular;
+  font-family: moon-regular, Arial, sans-serif;
```

Comment:
I think that we should introduce some global font styles... Leaving for discussion :)